### PR TITLE
[tensorflow][build][test] TFS 2.13 Currency

### PR DESCRIPTION
**Pull request by xncqr**
*GitHub Issue #, if available:*

**Note**: 
- If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

- All PR's are checked weekly for staleness. This PR will be closed if not updated in 30 days.

### Description
* Updating TFS to 2.13.0
* Updating GPU CUDNN to 8.9.3.28
* Updating TensorRT to 8.6.1.6

### Tests run
On commit [4efbf83](https://github.com/aws/deep-learning-containers/pull/3200/commits/4efbf8374a5682c0c5de25c080219d4e11de9fb3)
* Sanity Tests
* EC2 Tests
* ECS Tests
* EKS Tests

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: dlc_developer_config.toml
Comment (tejaschumbalkar) [inline dlc_developer_config.toml:1]: `graviton_mode` needs to be set to build graviton

Comment (xncqr) [reply dlc_developer_config.toml:1]: I'm not building graviton at this time.

Comment (xncqr) [reply dlc_developer_config.toml:1]: Graviton removed from PR

**Review by saimidu — COMMENTED** [review]
Dockerfiles and functional tests look good. Waiting for results of benchmark tests.

File: tensorflow/inference/docker/2.13/py3/cu118/Dockerfile.gpu [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+ && apt-get update \
+ && apt-get -y install --no-install-recommends \
+    nginx \
+    nginx-module-njs \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN ${PIP} install -U --no-cache-dir \
+    falcon==3.0.1 \
+    gunicorn==20.1.0
```
[the remark this answers is not in the document]
Comment (xncqr) [reply tensorflow/inference/docker/2.13/py3/cu118/Dockerfile.gpu:227]: Those are for the SM Tookit. there is work being done on the toolkit and will see if the can be done for the SM release.

**Review by saimidu — APPROVED** [review]

**Pull request merged**